modules/service/keeper/keeper_test.go

```
@@ -458,7 +458,7 @@ func (suite *KeeperTestSuite) TestKeeperRequestService() {
 
 	blockHeight := int64(1000)
 	ctx := suite.ctx.WithBlockHeight(blockHeight)
-	suite.app.BeginBlocker(ctx, abci.RequestBeginBlock{})
+	suite.app.BeginBlocker(ctx)
```

golangci-lint

Error return value of `suite.app.BeginBlocker` is not checked

(errcheck)